sormas-ui/src/main/java/de/symeda/sormas/ui/utils/CheckBoxTree.java

```diff
@@ -208,11 +212,9 @@ public void setValue(Map<ENUM, Boolean> newFieldValue) throws ReadOnlyException,
 	}
 
 	private void setToggleValues(Map<ENUM, Boolean> newFieldValue) {
-		settingToggles = true;
```

Comment:
removing the flag changes here makes `setValue` to be called recoursively for each checkbox, that again triggers setting the checkboxes and could result in inifite loop

the solution for the issue it to don't call setValue in the chaeckobx change listener while settingToggles is true.

Reply:
changed